sdk/keyvault/azure-security-keyvault-keys/src/main/java/com/azure/security/keyvault/keys/models/CreateRsaKeyOptions.java

```diff
@@ -9,13 +9,12 @@
 import java.util.Map;
 
 /**
- * Represents the configurable options to create an Rsa key.
+ * Represents the configurable options to create an RSA key.
  */
 @Fluent
 public class CreateRsaKeyOptions extends CreateKeyOptions {
```

Comment:
Missing method for `publicExponent` for 7.2. Secure Key Release (`exportable` and `securityPolicy` "properties") are in 7.3-preview.

Reply:
Will use this PR to remove 7.3-preview features too.

Reply:
By the way, [we do have](https://github.com/Azure/azure-sdk-for-java/blob/dc56d2058ca1d9152102f658ba3176b2109b3b42/sdk/keyvault/azure-security-keyvault-keys/src/main/java/com/azure/security/keyvault/keys/models/CreateRsaKeyOptions.java#L144) setters and getters for the public exponent of an RSA key.